spec/support/features/idv_helper.rb

```diff
@@ -126,4 +128,19 @@ def visit_idp_from_saml_sp_with_loa3
     @saml_authn_request = auth_request.create(settings)
     visit @saml_authn_request
   end
+
+  def expect_address_proofing_job
```

Comment:
WDYT of calling this `expect_address_proofing_job_enqueued`? Or just inlining it for clarity?